glenn2vcf/Makefile

```diff
@@ -0,0 +1,32 @@
+# Definitions
+build_output = runtime/glenn2vcf
+runtime_fullpath = $(realpath runtime)
+build_tool = runtime-container.DONE
+git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../glenn2vcf | cut -f1 -d " ")
+name = quay.io/ucsc_cgl/glenn2vcf
+tag = 2bb4bc2--${git_commit}
+
+# Steps
+build: ${build_output} ${build_tool}
+
+${build_output}: build/Dockerfile
+	cd build && docker build -t glenn2vcfbuild .
+	docker run -v ${runtime_fullpath}:/data glenn2vcfbuild cp /home/glenn2vcf/glenn2vcf /data
```

Comment:
Nit: `s/glenn2vcfbuild/glenn2vcf-build/g` for consistency throughout the repository.